about.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>About</title>
+    <link href="normalize.css" rel="stylesheet">
+    <link href="styles/about.css" rel="stylesheet">
+  </head>
+    <body>
+
+      <section class="header">
+        <li><a href="portfolio.html">portfolio.</a></li>
+        <li><a href="blog.html">blog.</a></li>
+        <li><a href="index(port).html">home.</a></li>
+        <li><a href="contact.html">contact.</a></li>
+      </section>
+
+      <div class="background">
+      <div class="info">
+        <h1>About Me</h1>
+        <div class="meta">
+          <a  href="https://i.imgur.com/AAfOSPx.jpg" target="_b" class="author"></a><br>
+        </div>
+      </div>
+    </div>
+
+    <section class="content">
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In nisl turpis, porttitor et finibus id, viverra a metus. Praesent non ante sed orci posuere varius quis sit amet dui. Cras molestie magna orci, id gravida dolor molestie in. Duis sollicitudin turpis quis tortor egestas, ut ultrices nisl elementum. Vestibulum sed ipsum eget nulla laoreet cursus in ac sem. Integer a suscipit justo, quis aliquam sapien. Maecenas et tellus nibh. Vivamus tincidunt eros id commodo pellentesque. porttitor et finibus id, viverra a metus. Praesent non ante sed orci posuere varius quis sit amet dui. Cras molestie magna orci, id gravida dolor molestie in. Duis sollicitudin turpis quis tortor egestas, ut ultrices nisl ele</p>
+      <!-- <p align="center"><a href="https://twitter.com/nodws" class="btn twtr" target="_b">Follow me on Twitter</a>
+      </p> -->
+    </section>
+<!--- Footer -------------->
+    <footer>
+      <ul class="footer">
+        <li>© 2018-</li>
+        <li>contact information: shelandebesai@gmail.com <a href="shelandebesai@gmail.com"</li>
```

Comment:
The anchor tag isn't closed, and it should read like:

`        <li>contact information: <a href="shelandebesai@gmail.com"> shelandebesai@gmail.com </a></li>`